it/src/test/java/com/linecorp/centraldogma/it/mirror/git/GitMirrorTest.java

```diff
@@ -180,6 +182,13 @@ void remoteToLocal() throws Exception {
         git.commit().setMessage("Add the release dates of the 'Infamous' series").call();
 
         mirroringService.mirror().join();
+        final List<Commit> commits = client.getHistory(projName, REPO_FOO, Revision.HEAD, Revision.INIT,
+                                                       PathPattern.all(), 1)
+                                           .join();
+        assertThat(commits).isNotEmpty();
+        final String detail = commits.get(0).detail();
+        assertThat(detail).isNotEmpty()
+                          .contains("Author", "Date");
```

Comment:
It would be helpful if we can assert the content of `Author` and `Data` to detect unexpected regressions in the future.

Reply:
and the original commit message as well, please.

Reply:
Thanks @trustin, @ikhoon, seem I see the concern:
1) From `RevCommit` we have `PersonIdent`, but on push, we still send the `MIRROR_AUTHOR = new Author("Mirror", "mirror@localhost.localdomain")` Is it correct?
2) If 1) was wrong, seems `Author` and `Date` are dynamic, so how can we assert the content?